### PR TITLE
Fix #97: Simplify change_version.

### LINKED ISSUE #97: ⚙️for-all-boosters: simplify change_version since we don't need parent support anymore
Take also the opportunity to remove the verification build since simply changing the booster version shouldn't impact the build.

### PR DESCRIPTION
**Pull request by metacosm**

### CODE REVIEW

File: for-all-boosters.sh [hunk 1]
```diff
@@ -247,58 +250,26 @@ change_version() {
...
-    targetParent=${2:-false}
-
-    # if we provide a 3rd arg, switch to parent processing instead
-    expr="project.version"
-    target="project"
-    if [ "${targetParent}" == true ]; then
-        expr="project.parent.version"
-        target="parent"
+    local newVersion=${1:-compute}
+    local jira
```
Comment (geoand) [inline for-all-boosters.sh]: Do we use the `jira` argument?

Comment (metacosm) [reply for-all-boosters.sh]: I have but I agree that it's kind of an outlier use case at this point…

Comment (geoand) [reply for-all-boosters.sh]: So do you want to keep it for the time being?

Comment (metacosm) [reply for-all-boosters.sh]: I'll remove it

Comment (geoand) [reply for-all-boosters.sh]: :+1:

**Pull request merged**